packages/sources/ignition-address-list/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@chainlink/ignition-address-list-adapter",
+  "version": "0.0.0",
```

Comment:
nit: should we make this v1?

Reply:
changeset will automatically bump it to v1